### PR TITLE
Run celery with the beat option in docker.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by johngian** [comment]
If the DB has no migrations (which usually hasn't by default) the `celery` container throws errors.

```
celery_1 | ProgrammingError: (1146, "Table 'mozillians.djcelery_periodictask' doesn't exist")

```

File: docker-compose.yml [hunk 1]
```diff
@@ -11,28 +11,28 @@ memcached:
...
   links:
     - db
     - es
     - broker
     - memcached
-  command: celery -A mozillians worker -l INFO
-  environment:
-    - C_FORCE_ROOT=true
-web:
+  command: bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
```
Comment (glogiotatidis) [inline docker-compose.yml]: tip:`migrate --noinput`

**Comment by akatsoulas** [comment]
@johngian / @glogiotatidis r?

**Comment by johngian** [comment]
r+ :ship:

**Pull request merged**